integration-tests/bats/export-tables.bats

```diff
@@ -308,3 +308,49 @@ SQL
    run dolt sql -q "SELECT * FROM timetable" -r csv
    [[ "$output" =~ "1,2021-06-02 15:37:24 +0000 UTC" ]] ||  false
 }
+
+@test "export-tables: parquet file export check with parquet tools" {
+    skiponwindows "Has dependencies that are missing on the Jenkins Windows installation."
+    dolt sql -q "CREATE TABLE test_table (pk int primary key, col1 text, col2 int);"
```

Comment:
Please add more types to this test. Try a datetime, a bool, and anyother wacky type you can think of.